docs/nipoppy/overview.md

```diff
@@ -1,39 +1,90 @@
 ## What is Nipoppy (formerly mr_proc)? 
-*Process long and prosper*
 
----
+[*Process long and prosper*](https://en.wikipedia.org/wiki/Vulcan_salute)
```

Comment:
I think we can remove this now :)

Reply:
Some people said they liked it, and I thought it was neat even though I didn't know it was from Star Trek. So I think we can keep it for now